src/lib/integration_test_cloud_engine/graphql_polling_log_engine.ml

```diff
@@ -56,7 +56,7 @@ let parse_event_from_log_entry ~logger log_entry =
       in
       event )
 
-let rec poll_get_filtered_log_entries_node ~logger ~event_writer
+let rec get_filtered_log_entries_of_node ~logger ~event_writer
```

Comment:
`of_node` seems redundant, given that it's passed a node as its only positional argument.